docs/reference/commandline/volume_inspect.md

```diff
@@ -25,7 +25,7 @@ Options:
       --help            Print usage
 ```
 
-Returns information about a volume. By default, this command renders all results
+Return information about a volume. By default, this command renders all results
```

Comment:
Sorry，it is  careless operation. I want to restart CI when experimental is failed.  I checked it and update the PR.